helm/ovn-kubernetes/templates/ovn-setup.yaml

```diff
@@ -63,3 +63,15 @@ kind: Namespace
 metadata:
   name: {{ $hostNetworkNamespace }}
 {{- end }}
+
+{{- if (and (hasKey .Values.global "dockerConfigSecret") .Values.global.dockerConfigSecret.create .Values.global.imagePullSecretName) }}
```

Comment:
can you please simplify this?

{{- if (and .Values.global.dockerConfigSecret .Values.global.dockerConfigSecret.create) }}

Reply:
for sure!